release_images.yml

```diff
@@ -2,182 +2,45 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "huggingface_pytorch"
-    version: "1.10.2"
-    arch_type: "x86"
-    hf_transformers: "4.17.0"
-    training:
-      device_types: [ "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
-    inference:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
+    framework: "tensorflow"
+      version: "2.7.0"
+      arch_type: "x86"
+      customer_type: "sagemaker"
+      inference:
+        device_types: ["gpu"]
```

Comment:
Are we not releasing the cpu images for this? I read the PR description, just wanted to ensure

Reply:
We are releasing just TF inference SM GPU DLCs.(Ref PR - https://github.com/aws/deep-learning-containers/pull/2149 Ticket Ref- https://t.corp.amazon.com/P69594461/overview)